### ISSUE TITLE
After watching the video about work with psd templates

### ISSUE THREAD

**Issue by Gavolexl** [avocode/avocode#945]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.8.2
Source UID: 74644
```
Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.
    at invariant (C:\Users\Gavole\AppData\Local\avocode\app-2.8.2\resources\app.asar\node_modules\flux\node_modules\fbjs\lib\invariant.js:35:15)
    at Dispatcher.dispatch (C:\Users\Gavole\AppData\Local\avocode\app-2.8.2\resources\app.asar\node_modules\flux\lib\Dispatcher.js:167:160)
    at e.dispatch (file:///C:/Users/Gavole/AppData/Local/avocode/app-2.8.2/resources/app.asar/dist/avocode.min.js:21:12406)
    at t.e.dispatch (file:///C:/Users/Gavole/AppData/Local/avocode/app-2.8.2/resources/app.asar/dist/avocode.min.js:2:20975)
    at t.focusWindow (file:///C:/Users/Gavole/AppData/Local/avocode/app-2.8.2/resources/app.asar/dist/avocode.min.js:15:26701)
    at e._handleWindowFocus (file:///C:/Users/Gavole/AppData/Local/avocode/app-2.8.2/resources/app.asar/dist/avocode.min.js:21:23633)
    at file:///C:/Users/Gavole/AppData/Local/avocode/app-2.8.2/resources
```

**Comment by jakubzitny** [comment]
Hi @Gavolexl, thanks for reporting this. This has happened to few users already and we're already working on a fix. It should be ready for the next patch version some time next week.

**Comment by petrbrzek** [comment]
Fixed in 2.8.3.
